### PR TITLE
👷 Try to simplify build.yml

### PR DESCRIPTION
**Pull request by falexwolf**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -14,63 +14,37 @@ jobs:
...
-          cache-dependency-path: ".github/workflows/build.yml" # See dependencies below
-
-      - name: Install Nextflow
-        uses: nf-core/setup-nextflow@v1
+          cache-dependency-path: ".github/workflows/build.yml"
+      - uses: nf-core/setup-nextflow@v1
         with:
           version: "latest-stable"
-
       - name: Cache Nextflow output
```
Comment (falexwolf) [inline .github/workflows/build.yml:37]: I wasn't able to figure out whether this is actually used or not. It seems like the nextflow pipeline is actually running, doesn't it?

I do think that an automatic caching by-passing the run would make a lot of sense to better iterate on this use case. 

The CI times are waaaaaaaaay tooooooooo loooooong to iterate fast.

(I wasn't keen on installing nextflow on my laptop...)

**Comment by falexwolf** [comment]
For whatever reason, this one ran 30 min, the other PR only ran 22 min.

File: .github/workflows/build.yml (hunk 1 above)
Comment (Zethson) [reply .github/workflows/build.yml:37]: Yeah I couldn't get it to work. I'm pretty sure that this is the output folder and I expected the `nextflow resume` command to pick it up, but for some reason it doesn't.

**Review by Zethson — APPROVED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (falexwolf) [reply .github/workflows/build.yml:37]: Then let's just remove this from the build.yml?

Comment (falexwolf) [reply .github/workflows/build.yml]: Did so while resolving the conflict!

**Pull request merged**